documentation-support/src/main/java/tech/picnic/errorprone/documentation/BugPatternExtractor.java

```diff
@@ -91,7 +94,36 @@ private static <T extends AnnotationValue> T doCast(AnnotationValue value, Class
   }
 
   @AutoValue
+  @JsonDeserialize(as = AutoValue_BugPatternExtractor_BugPatternDocumentation.class)
   abstract static class BugPatternDocumentation {
+    static BugPatternDocumentation create(
```

Comment:
Is there an order we are following for these parameters ?

Is it worth the hassle to find a better way to sort them ? Orthographicall or Objects > primities > lists for example ?

Reply:
The order we're following here matches the order in which the properties are defined. Those in turn are from "high level"/"most important" to "minor"/"more of a detail". It's not an exact science :smile:. Ordering them by type or name would IMHO not improve readability.